include/z64save.h

```diff
@@ -770,7 +770,7 @@ typedef enum LinkAge {
 #define INFTABLE_AB 0xAB
 #define INFTABLE_B0 0xB0
 #define INFTABLE_B1 0xB1
-#define INFTABLE_B4 0xB4
+#define INFTABLE_TALKED_TO_GORON_MSG_3054 0xB4
```

Comment:
(I feel like these names should be more specific than `INFTABLE_TALKED_TO_GORON_MSG_3054`, but it's not worse than before so it's whatever)

Reply:
yeah, probably i should shuffle through the messages more closely

Reply:
ok, gotcha